src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -71,6 +71,13 @@
     public $incrementing = true;
 
     /**
+     * Sets the type used by the ID.
```

Comment:
We should make it clear that this property is ignored when incrementing is not enabled.